ui/raidboss/data/07-dt/trial/zoraal-ja-ex.txt

```diff
@@ -0,0 +1,122 @@
+# -r 4zpQDykmMGJnX2FW -rf=8
+# Sorry, no `-ii` parameters, I cleaned this up by hand because it was such a mess
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# Forward Half:
+# 937B - Right sword glowing, safe spot is front+right
+# 999B - Left sword glowing, safe spot is front+left
+
+# Forward Half:
+# 937D - Right sword glowing, safe spot is back+left
+# 937E - Left sword glowing, safe spot is back+right
+
+# Arena phase
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+3.2 "--sync--" Ability { id: "93A7", source: "Zoraal Ja" } window 3.2,1
+8.0 "--sync--" StartsUsing { id: "9398", source: "Zoraal Ja" } window 8,8
+12.7 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+23.0 "Multidirectional Divide (cast)" Ability { id: "93A2", source: "Zoraal Ja" }
+34.0 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+35.0 "Half Full" #Ability { id: "939E|9380", source: "Zoraal Ja" }
+35.0 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
```

Comment:
Maybe add a substitution, e.g. 'Forward/Backward Edge' to `timelineReplace` (repeated elsewhere below).